google/cloud/firestore_v1/order.py

```diff
@@ -29,7 +29,8 @@ class TypeOrder(Enum):
     REF = 6
     GEO_POINT = 7
     ARRAY = 8
-    OBJECT = 9
+    VECTOR = 9
+    OBJECT = 10
```

Comment:
The comment says "# NOTE: This order is defined by the backend and cannot be changed." Isn't this changing OBJECT from 9 to 10? How is that handled on the backend? Is it a breaking change?

Reply:
From API perspective, it is map_value. In backend we do implement it as a new type (so that we can do things like storage optimization)